### ISSUE TITLE
KeyError: b'KEY'

### ISSUE THREAD

**Issue by andreas-zeller** [avibrazil/iOSbackup#12]
This is for iOSbackup-0.9.912 from pip.

Extracting an encrypted backup from iOS 14.5.1, I get the error

```
KeyError: b'KEY'
```

raised in `unwrapKeyForClass(self, protection_class, persistent_key)`:

```python
ck = self.classKeys[protection_class][b"KEY"]
```

The element `self.classKeys[protection_class]` has keys 

```
b'UUID', b'CLAS', b'WRAP', b'KTYP', b'WPKY'
```

but no `b'KEYS'`.

Opening an older backup from iOS 14.4 still works fine.

**Comment by dkg** [comment]
This error message typically is associated with having the wrong password for the decryption.

My understanding is that the `b'WPKY'` member is the "wrapped key", and if the key unwrap works, then the dict gets populated with a `b'KEY'` member.  If the key unwrap doesn't work, then failure happens because the `b'KEY'` member isn't there.

I agree that the error handling could be more explicit here though!

**Comment by avibrazil** [comment]
It happened to me too. Probably because I changed to a new Mac or got a new iPhone (can't remember the cause). I had to start over with iOSbackup with a new clear text password.

I'd like to improve error handling and user feedback, but I don't know how to precisely detect that the password is the problem. Just to check if `b'KEY'` is present doesn't seem very precise. Suggestions of a strategy and actual PRs are welcome.
